twitter.css

```diff
@@ -0,0 +1,284 @@
+* {
```

Comment:
Also very organised CSS

Reply:
I liked that your CSS did not go on forever, unlike mine lol!
Can add some comments also, just so it is easier to find section quickly.